ColorPatterns/Scenes/StartMenu/StartMenuController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  StartMenuController.swift
+//  ColorPatterns
+//
+//  Created by Marek Skrzelowski on 23/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+class StartMenuController: UIViewController {
+
+    @IBOutlet private var levelLabel: [SharedButton]!
+
+    let viewModel: StartMenuViewModel
+
+    init(viewModel: StartMenuViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/76417ef6d6c61cd410bc6d0c2f91dc2e04d78290/ColorPatterns/Scenes/StartMenu/StartMenuController.swift#L28"></span>Override methods which only call super can be removed</td>
    </tr>
  </tbody>
</table>